pkg/metaserver/util/util.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"strings"
 
+	pluralize "github.com/gertd/go-pluralize"
```

Comment:
Looks good, but it's better to keep same with Kubernetes. Does K8s also use this pkg?

Reply:
It's better to keep same with K8s.

Reply:
@wbc6080 thanks for taking care of this. this is tricky but annoying problem.

as @fisherxu mentions, probably we would want to go with k8s approach for now. but your concern also stands, we would want keep the note for future possible problems though.

Reply:
I have made modifications based on the processing method in k8s, and also retained the original issue as a reference.